vhdbuilder/packer/cleanup.sh

```diff
@@ -44,6 +44,28 @@ if [ -n "$id" ]; then
   az storage account delete -n ${SA_NAME} -g ${AZURE_RESOURCE_GROUP_NAME} --yes
 fi
 
+#clean up storage account created over X hours ago (testing; X=19)
+# set subscription to AKS VHD Image Release
+az account set -s a15c116e-99e3-4c59-aebc-8f864929b4a0
+AZURE_RESOURCE_GROUP_NAME="akswinvhdbuilderrg"
+EXPIRATION_IN_HOURS=20
```

Comment:
The setting does not match the value in PR's title and description (18)

Reply:
Sorry about the confusion. It was meticulously set to delete accounts manually created X hours ago as a trial run. The time had to be precise, and so the inconsistency comes from the time delay. Will change to 24*7. Or better yet, if there is no gap in completely cleaning the storage account upon cancellation, the manual deletion is not necessary.